custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -265,11 +268,68 @@ def predict(self, data, model=None, **kwargs):
 
         return predictions
 
+    def try_to_fit_on_custom_wrapper(self, X, y, output_dir):
```

Comment:
And just the last effort...
rename this to: `_drum_autofit`

Reply:
why

Reply:
1. because `try_to_fit_on_custom_wrapper` name doesn't reflect the actions done inside...`_drum_autofit_internal` would be even better
2. I'd also rename `wrapped_object` to `marked_object`
3. replace `custom()` in the comment with `drum_autofit()`
4. names `drum_autofit()` and `_drum_autofit_internal` form together a pair, so it is easy to understand they are used together.